TLM/TLM/Manager/Impl/OptionsManager.cs

```diff
@@ -230,10 +269,15 @@ public class OptionsManager
 
                 save[58] = (byte)Options.SavegamePathfinderEdition;
 
+                Options.Available = true;
+
                 return save;
             }
             catch (Exception ex) {
                 ex.LogException();
+
+                Options.Available = true;
```

Comment:
same here

Reply:
Yes. If save method fails, the option field values in the `Options.cs` are still intact and TM:PE is still operable.

Reply:
Removed `Options.Available` stuff from `SaveData()` as that method doesn't alter the option field values.